### PR TITLE
Versions Roles dual-list

### PR DESCRIPTION
**Pull request by ghost**
Create dual-list container and use for all dual list screens
Refactor confirm modal

### CODE REVIEW

File: client/src/styles/app.css [hunk 1]
```diff
@@ -51,7 +51,7 @@ a.disabled {
 }
 
 .options-large {
-  width: 265px;
+  width: 420px;
```
Comment (ghost) [inline client/src/styles/app.css:54]: @ssavery-ib  why?

File: client/src/app/journey/journey.service.ts [hunk 2]
```diff
@@ -47,16 +45,15 @@ export class JourneyService {
...
-    .map(response => response.json());
+    return this.http.get(this.config.Server + 'nsdc/v1.0/journeys/versions', { headers: this.headers })
+    .map(res => res.json() as JourneyVersion[]);
   }
 
-  getVersionById(id) {
+  getVersionsByRole(roleId) {
     this.headers.set('X-TOKEN', this.loginService.getSessionId());
-    let url = this.config.Server + 'nsdc/v1.0/journeys/versions/' + id;
+    let url = this.config.Server + 'nsdc/v1.0/journeys/versions/roles/' + roleId;
```
Comment (ssavery-ib) [inline client/src/app/journey/journey.service.ts:56]: This always bugged me, why are we creating a variable here, let along a let ... We can look at this when we refactor the services.

File: client/src/styles/app.css (hunk 1 above)
Comment (ssavery-ib) [reply client/src/styles/app.css:54]: I added a new button, Journey Versions

Comment (ghost) [reply client/src/styles/app.css:54]: fair enough

**Pull request merged**